packages/lesswrong/themes/userThemes/darkMode.ts

```diff
@@ -46,7 +46,11 @@ export const invertedGreyscale = {
 };
 
 const greyAlpha = (alpha: number) => `rgba(255,255,255,${alpha})`;
-const inverseGreyAlpha = (alpha: number) => `rgba(0,0,0,${alpha})`;
+
+const inverseGreyAlpha = (alpha: number) => {
+  const [r, g, b] = invertColor([1, 1, 1, 1]);
+  return `rgba(${zeroTo255(r)},${zeroTo255(g)},${zeroTo255(b)},${alpha})`;
+}
```

Comment:
This change is needed because we apply some biasing when convert shades of grey into dark mode to make the contrast more attractive, however, the biasing isn't currently applied to this function so the colors it returns are slightly wrong.